ImplementationGuide/markdown/Katalog/CodeSystem_Interaktionen.md

```diff
@@ -1,56 +1,23 @@
 ### Interaktionen
 
-Für die Ressource CodeSystem MUSS die REST-Interaktion "READ" implementiert werden.
+Für die Ressource CodeSystem SOLL die REST-Interaktion "READ" implementiert werden.
 
 Folgende Suchparameter sind für das Bestätigungsverfahren relevant, auch in Kombination:
 
-1. Der Suchparameter "_id" MUSS unterstützt werden:
+1. Der Suchparameter "_id" SOLL unterstützt werden:
```

Comment:
issue (blocking): auch wenn das Profil als ganzes optional ist, sind es diese Suchparameter nicht (unter der Voraussetzung dass das Profil umgesetzt wird). Somit würde ich hier MUSS statt SOLL behalten. Ansonsten kann man die Suchparameter als optional interpretieren und somit bei der Umsetzung vom CodeSystem ggf. weglassen

Reply:
@alexzautke question: wie sinnvoll ist eigentlich der Suchparameter _id überhaupt? In welchem Szenario würde man nach einem CodeSystem mittels _id statt url suchen? Könnte man dann nicht etwa einen GET auf CodeSystem/_ID_ dafür absetzen, wenn man schon anhand einer ID zugreifen möchte?

Reply:
@alexey-tschudnowsky Die FHIR-Core-Spec sagt, dass _id immer unterstützt werden muss: https://www.hl7.org/fhir/r4/search.html#conformance

Reply:
"Servers can choose which parameters to support (other than _id above)."